### DISCUSSION TITLE
Images are uploaded with incorrect content type (application/octet-stream) to Azure Blob storage

### DISCUSSION THREAD

**Discussion by superhp** [directus/directus#8292] [Archive]
When the image is uploaded from Directus to Azure Blob storage, its content type is always set to application/octet-stream. Instead, it should be set as image/svg+xml or image/jpeg. Otherwise, we can't use the image URL from the Azure Blob storage in img element. In such case, the image is automatically downloaded as a file. Is it a bug or can it be configured somehow?

**Comment by joselcvarela** [comment]
Hello,
It seems it is an issue.
I have created a PR to solve it https://github.com/directus/directus/pull/8294

**Comment by superhp** [comment]
Thank you very much for really quick response and fix! Any ideas on when it should be released?

**Reply by rijkvanzanten** [reply]
It's already merged, so it'll be in the next rc release 👍🏻
